src/cluster/theory.rst

```diff
@@ -34,10 +34,17 @@ thereby override the defaults in ``default.ini``.
 
 The number of copies of a document with the same revision that have to be read
 before CouchDB returns with a ``200`` is equal to a half of total copies of
-the document plus one. It is the same for the number of nodes that need to
-save a document before a write is returned with ``201``. If there are less
-nodes than that number, then ``202`` is returned. Both read and write numbers
-can be specified with a request as ``r`` and ``w`` parameters accordingly.
+the document plus one *in the current partition*. That is, if there is a network
```

Comment:
"partition" is not a word common to CouchDB parlance. Do you mean node? shard?

Reply:
I was trying to describe a network partition. Specifically, here I'm trying to say that if there's a network partition then the quorum is relative to only the subset of nodes on the side of the partition where the client is.

Definitely open to different wording if this is confusing.

Reply:
Ah, now I see what you're getting at.  That is an important point. And the paragraph was already a bit confusing before your addition. Perhaps a rewrite of the entire section is in order.

Example:

> In clustered operation, a quorum must be reached before CouchDB returns a ``200`` for a fetch, or ``201`` for a write operation. A quorum is defined one plus half the number of copies.

> For read operations, the maximum number of copies is the number of currently-accessible nodes, meaning that in the case of a failure or network partition, the effective number of copies may be lower.  The number of read copies can be set with the ``r``parameter.

> For write operations, the number of copies can be set using the ``w`` parameter. If fewer than this number of nodes is available, a ``202`` will be returned.

Reply:
Overall I like what you have better. Will start with that but make a few changes to be more explicit. Updated PR coming soon...